tx-builder/src/ckb/utils/script_utils.rs

```diff
@@ -0,0 +1,36 @@
+use ckb_sdk::unlock::OmniLockConfig;
+
+use ckb_types::core::ScriptHashType;
+use ckb_types::packed::Script;
+use ckb_types::prelude::{Builder, Entity, Pack};
+use ckb_types::{H160, H256};
+
+use hex_literal::hex;
+
+// reference : https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0042-omnilock/0042-omnilock.md#notes
+static OMNILOCK_CODE_HASH_MIRANA: [u8; 32] =
+    hex!("9b819793a64463aed77c615d6cb226eea5487ccfc0783043a587254cda2b6f26");
```

Comment:
use `h256!("0x9b819793a64463aed77c615d6cb226eea5487ccfc0783043a587254cda2b6f26")` is enough, [h256](https://docs.rs/ckb-types/0.110.0/ckb_types/macro.h256.html)